### ISSUE TITLE
How to upgrade to latest devtools?

### ISSUE THREAD

**Issue by arcticfox88** [flutter/devtools#5008]
My Flutter plugin in android studio has been upgraded to the latest, but the version of the devtools tool is only 2.15.0.

My Flutter SDK:
```
Flutter 3.3.10 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 135454af32 (3 weeks ago) • 2022-12-15 07:36:55 -0800
Engine • revision 3316dd8728
Tools • Dart 2.18.6 • DevTools 2.15.0
```

**Comment by polina-c** [comment]
DevTools and Dart SDK have different versioning. It is coincident that they are so close to each other.
Your Flutter environment is intact and ready to create amazing applications.

**Comment by arcticfox88** [comment]
@polina-c 
The `devtools` tool has been released with version 2.20.0, can't I proactively upgrade it?
Is there a binding between `devtools` and the Dart SDK version? It is not clearly stated here. How exactly do I actively upgrade to use the latest `devtools` tool?

**Comment by kenzieschmoll** [comment]
DevTools is shipped as part of the Dart SDK, therefore shipped as part of the Flutter SDK as well. To get the newest DevTools features, much like the newest Flutter features, you'll have to upgrade your Flutter version. The latest shipped DevTools will always be available on the Flutter master branch, and the DevTools version on the Flutter beta branch will be newer than the DevTools version on the Flutter stable branch. Hope that helps!
